doc/usecases/membership_application.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: Membership Application
+
+## Summary
+
+- **Scope:** Membership
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can apply for a membership (account)
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and is not logged in.
+- **Main success scenario:**
+  - The user can click an register button which leads them to the apply for membership page
+  - The user can fill out a form with their username and email address
+  - The user can apply for a membership
```

Comment:
We should add here, that there are two kinds of membership the user can apply for:

- one with a yearly paid subscription and
- one free membership the user can maintain via gathering a certain number of points per year.

These points can be obtained by completing Blossoms in the Expert Track (#105 gamification/gain_blossoms.md) and engaging with other users in meaningful ways (e.g. writing good reviews).